packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditTables.tsx

```diff
@@ -3,10 +3,11 @@ import {
   RecordPatch,
   DataTable,
   useColumns,
-  NumberInputCell,
   CurrencyInputCell,
   useTranslation,
   getExpiryDateInputColumn,
+  NonZeroNumberInputCell,
+  PositiveNumberInputCell,
```

Comment:
should these actually technically be `NonNegative`  ( >= 0 ) and `Positive` ( > 0 ) 🤣

Reply:
I've always thought of `Positive` as including `0`
These guys agree with you though: https://math.stackexchange.com/questions/18464/is-positive-the-same-as-non-negative. I shall update.